### PR TITLE
WICKET-6055 non-blocking lazy loading

### PR DESCRIPTION
**Pull request by svenmeier**
Building on #151 I've built this alternative implementation with the following advantages:

- lazy loading can now *start* on an Ajax request too
- much simpler without using the event bus
- the timer adjusts to the minimum of the preferred timeout of all LazyLoadPanels
- reworded methods and improved JavaDoc

### CODE REVIEW

**Comment by dashorst** [comment]
Looks good to me...

**Comment by svenmeier** [comment]
I've improved compatibility with the previous implementation. Any objections against merging this into Wicket 8?

File: wicket-extensions/src/test/java/org/apache/wicket/extensions/markup/html/AjaxLazyLoadPanelTesterTest.java [hunk 1]
```diff
@@ -38,12 +38,12 @@
...
 	public void test()
 	{
-		AjaxLazyLoadPanel panel = new AjaxLazyLoadPanel("panel")
+		AjaxLazyLoadPanel<Component> panel = new AjaxLazyLoadPanel<Component>("panel")
 		{
 			private static final long serialVersionUID = 1L;
 
 			@Override
-			public Component getLazyLoadComponent(final String markupId)
+			protected Component getLazyLoadComponent(String markupId)
```
Comment (solomax) [inline wicket-extensions/src/test/java/org/apache/wicket/extensions/markup/html/AjaxLazyLoadPanelTesterTest.java]: I guess `markupId` can remain `final` here ...

Comment (svenmeier) [reply wicket-extensions/src/test/java/org/apache/wicket/extensions/markup/html/AjaxLazyLoadPanelTesterTest.java]: Thanks, it should still be public too.

**Comment by dashorst** [comment]
Go ahead!

**Pull request closed without merging**